src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        setColor(color);
+        obtainTheArea();
+    }
+
+    public void obtainTheArea() {
+        setArea(firstLeg * secondLeg / 2);
+    }
+
+    @Override
+    public String draw() {
```

Comment:
The 'draw' method should not be responsible for calculating the area. It should only return the string representation of the figure. Consider calculating the area before calling 'draw' if needed.